[EDU-1655] - Pub/Sub Presence example (#2325)
ably/docs: examples/pub-sub-presence/page.md

# Presence updates with Pub/Sub

Presence enables users to see who else is online within an application.

> Use presence to display which users are currently online in a channel. Users can also manually update their status, such as to set themselves as 'Away', or write a custom message.

Reviewer: ```suggestion
Use presence to display which users are currently 'present' on a channel. Users can also manually update their status, such as to set themselves as 'Away', or write a custom message.
```

Presence enables users to quickly understand the availability of others to communicate more effectively. For example, on a live video stream users can see who else is currently viewing.

> Presence is implemented using [Ably Pub/Sub](https://ably.com/docs/products/channels). The Ably SDK provides features such as presence, publishing and subscribing to messages within channels in a high performance and scaling capability.

Reviewer: Suggest we replace the 2nd sentence with the "agreed" upon one for Pub/Sub 🙂
